official/transformer/v2/transformer_main_test.py

```diff
@@ -75,23 +85,26 @@ def test_train_static_batch(self):
     t = tm.TransformerTask(FLAGS)
```

Comment:
does this test pass? for me it was failing with some weird reasons

Reply:
Yes, it fails in TF 1.0 on GPU, but is not run in our test because it is CPU only.  I think because of that input problem.  I have another PR coming where all GPU tests pass on TF 1.0 where I flags this as TF 2.0 only.  And then I will have a PR where everything works on TF 2.0 CPU/GPU.  I guess is ~6 days as I now have a plan (P100s for now) to spin up 2 GPU systems for the presubmits.